app/models/work_activity.rb

```diff
@@ -80,20 +134,69 @@ def file_changes_html
                end
         "<tr><td>#{icon}</td><td>#{change['action']}</td> <td>#{change['filename']}</td>"
       end
-      "<p><b>Files updated:</b></p><table>#{changes_html.join}</table>"
+
+      children = "<p><b>Files updated:</b></p><table>#{changes_html.join}</table>"
+      event_html(children: children)
     end
 
     # Returns the message formatted to display _metadata_ changes that were logged as an activity
     def metadata_changes_html
       text = ""
       changes = JSON.parse(message)
+
       changes.keys.each do |field|
-        values = changes[field].map { |value| change_value_html(value) }.join
-        text += "<p><b>#{field}</b>: #{values}</p>"
+        change = changes[field]
+        mapped = change.map { |value| change_value_html(value) }
+        values = mapped.join
```

Comment:
Should we in general avoid method chaining?

Reply:
I apologize, my approach has been to restructure these out of habit (I have attempting to follow the guidance of Sandi Metz: https://sandimetz.com/). I can revert these changes.